examples/methods/sha/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "methods-guest"
```

Comment:
I think you can use `src/main.rs` if also adjust the name of this crate so that it matches the binary name you expect.